projects/components/src/textarea/textarea.component.ts

```diff
@@ -74,6 +83,10 @@ export class TextareaComponent implements ControlValueAccessor, OnInit {
     this.propagateControlValueChangeOnTouch = onTouch;
   }
 
+  public setDisabledState(isDisabled?: boolean): void {
+    this.disabled = isDisabled ?? false;
+  }
```

Comment:
you want to add cdr markForCheck here?

Reply:
It seems not necessary to put it here for this component